samples/openthread/ncp/CMakeLists.txt

```diff
@@ -5,13 +5,9 @@
 #
 cmake_minimum_required(VERSION 3.13.1)
 
-set(NRF_SUPPORTED_BOARDS
-	nrf52840dk_nrf52840
-	nrf52833dk_nrf52833
-)
-
 set(OT_NCP_VENDOR_HOOK_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
+list(APPEND OVERLAY_CONFIG ${ZEPHYR_NRF_MODULE_DIR}/samples/openthread/common/overlay-ot-defaults.conf)
```

Comment:
Sorry on this.
This actually has to be reverted back to 
```
list(APPEND OVERLAY_CONFIG ../common/overlay-ot-defaults.conf)
```
because `${ZEPHYR_NRF_MODULE_DIR}` is not defined until `find_package(Zephyr)` has executed :disappointed: 

Will see if this can be improved upstream, but for this PR, `..` has to be used.